terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1732,6 +1768,35 @@ def qq(*words):
                     'Owner': config.owner
                 }
             }
+        },
+        'aws_cloudwatch_metric_alarm': {
+            **{
+                'gitlab_' + resource: {
+                    'alarm_name': 'azul-gitlab_' + resource,
+                    'comparison_operator': 'GreaterThanOrEqualToThreshold',
+                    'evaluation_periods': period,
+                    'period': 60 * 10,
+                    'metric_name': metric,
+                    'namespace': 'CWAgent',
+                    'dimensions': dimensions | {
+                        'InstanceId': '${aws_instance.gitlab.id}',
+                    },
+                    'statistic': stat,
+                    'threshold': threshold,
+                    'treat_missing_data': 'missing',
+                    **{
+                        state + '_actions': ['${data.aws_sns_topic.monitoring.arn}']
+                        for state in ('insufficient_data', 'alarm', 'ok')
+                    },
+                } for resource, period, metric, dimensions, stat, threshold in
```

Comment:
```suggestion
                } for resource, metric, periods, stat, threshold, dimensions in
```

and then reorder the tuple elements below accordingly.